### PR TITLE
add kustomize plugin

### PR DESCRIPTION
**Pull request by rubbish**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ place for people (and asdf itself) to look for plugins.
 | Kubeseal   | [stefansedich/asdf-kubeseal](https://github.com/stefansedich/asdf-kubeseal) | [![Build Status](https://travis-ci.com/stefansedich/asdf-kubeseal.svg?branch=master)](https://travis-ci.com/stefansedich/asdf-kubeseal)
 | Kubesec   | [vitalis/asdf-kubesec](https://github.com/vitalis/asdf-kubesec) | [![Build Status](https://travis-ci.org/vitalis/asdf-kubesec.svg?branch=master)](https://travis-ci.org/vitalis/asdf-kubesec)
 | Kubeval   | [stefansedich/asdf-kubeval](https://github.com/stefansedich/asdf-kubeval) | [![Build Status](https://travis-ci.com/stefansedich/asdf-kubeval.svg?branch=master)](https://travis-ci.com/stefansedich/asdf-kubeval)
+| Kustomize | [Banno/asdf-kustomize](https://github.com/Banno/asdf-kubectl) | [![Build Status](https://travis-ci.com/Banno/asdf-kustomize.svg?branch=master)](https://travis-ci.com/Banno/asdf-kustomize)
```
Comment (Stratus3D) [inline README.md]: This link is wrong. It points to asdf-kubectl.

File: plugins/kustomize [hunk 2]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/Banno/asdf-kustomize
```
Comment (Stratus3D) [inline plugins/kustomize]: This needs to be the URL to the Git repository, so it should end with `.git`.

**Comment by rubbish** [comment]
@Stratus3D I've made those fixups. Sorry about that. Thanks!

**Comment by Stratus3D** [comment]
Thanks!

**Comment by gdlx** [comment]
Hi @lamdor!
It seems nobody maintains this plugin at Banno anymore and an old PR is needed to fix a bad variable name choice: https://github.com/Banno/asdf-kustomize/pull/12

Are you in contact with the repo owners to ask for a quick review ? Or do you know if some fork could replace the forsaken one? Maybe @log2's  https://github.com/log2/asdf-kustomize/tree/master ?

**Pull request merged**